### ISSUE TITLE
Backend-exemplar needs a DB instance

### ISSUE THREAD

**Issue by nam20485** [hackoregon/backend-examplar-2018#94]
Need a DB instance to run unit tests against.

@MikeTheCanuck

**Comment by znmeb** [comment]
What kind of "instance"? A test database on AWS?

**Comment by nam20485** [comment]
Yes, like the team backend DBs. So that the backend can connect using our standard env. vars and pytest can run against. The schema would only need to be what is initially created to support the basic schema used by the Django app (application and songs api).

**Comment by nam20485** [comment]
If we want the backend exemplar to be a good starting point I think that at the least it should support the DB testing and travis/ECR deployment that we use in production, even if we don't go all the way and deploy a live instance to ECS.

That way when they spawn a new backend it is ready for our production deploy infrastructure and they don't have to set that up themselves.
